src/encoded/types/shared_calculated_properties.py

```diff
@@ -232,97 +229,10 @@ def biosample_summary(self,
-                                    if gm_object.get('introduced_tags_array'):
-                                        modification_dict['tags'] = []
-                                        for tag in gm_object.get('introduced_tags_array'):
-                                            tag_dict = {'location': tag['location']}
-                                            if tag.get('promoter_used'):
-                                                tag_dict['promoter'] = request.embed(
-                                                    tag.get('promoter_used'),
-                                                            '@@object').get['label']
-                                            modification_dict['tags'].append(tag_dict)
-
-                                    modifications_list.append((gm_object['method'], modification_dict))
-
-                            preservation_method = None
-                            dictionary_to_add = generate_summary_dictionary(
-                                request,
-                                organismObject,
-                                donorObject,
-                                biosampleObject.get('age'),
-                                biosampleObject.get('age_units'),
-                                biosampleObject.get('life_stage'),
-                                biosampleObject.get('sex'),
-                                biosampleTypeObject.get('term_name'),
-                                biosampleTypeObject.get('classification'),
-                                biosampleObject.get('starting_amount'),
-                                biosampleObject.get('starting_amount_units'),
-                                biosampleObject.get('depleted_in_term_name'),
-                                biosampleObject.get('phase'),
-                                biosampleObject.get('subcellular_fraction_term_name'),
-                                biosampleObject.get('synchronization'),
-                                biosampleObject.get('post_synchronization_time'),
-                                biosampleObject.get('post_synchronization_time_units'),
-                                biosampleObject.get('post_treatment_time'),
-                                biosampleObject.get('post_treatment_time_units'),
-                                biosampleObject.get('post_nucleic_acid_delivery_time'),
-                                biosampleObject.get('post_nucleic_acid_delivery_time_units'),
-                                treatment_objects_list,
-                                preservation_method,
-                                part_of_object,
-                                originated_from_object,
-                                modifications_list,
-                                True)
-
-                            dictionaries_of_phrases.append(dictionary_to_add)
+                        biosample_dict = biosample_summary_dictionary(request, biosampleObject)
```

Comment:
update the function name if you change it according to comments above

Reply:
Updated